core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -548,6 +543,8 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |  --conf, -c PROP=VALUE       Arbitrary Spark configuration property.
         |  --properties-file FILE      Path to a file from which to load extra properties. If not
         |                              specified, this will look for conf/spark-defaults.conf.
+        |  --load-spark-defaults       Whether to load properties from conf/spark-defaults.conf,
+        |                              even if --properties-file is specified.
```

Comment:
I think it is good to mention which takes precedence when both are specified.

Reply:
Good point. Added.